ocaml/libs/tracing/tracing.ml

```diff
@@ -684,9 +684,9 @@ module Export = struct
           let headers =
             Cohttp.Header.of_list
               [
-                ("accepts", "application/json")
-              ; ("content-type", "application/json")
-              ; ("content-length", string_of_int (String.length body))
+                ("Content-Type", "application/json")
+              ; ("Content-Length", string_of_int (String.length body))
+              ; ("Host", "jaeger.xenrt.citrite.net:9411")
```

Comment:
Why is the host hardcoded? Is this overwritten in case other hostnames are used?